db.dataview/test/unit/src/org/netbeans/modules/db/dataview/util/TestCaseDataFactory.java

```diff
@@ -39,11 +39,11 @@
     public static  String DB_SQLINSERT="dbinsert.sql";
     public static String DB_SQLSELECT="dbselect.sql";
     public static String DB_SQLUPDATE="dbupdate.sql";
-    public static  String DB_TEXT= "dbdata.txt";
+    public static  String DB_DATA= "dbdata.properties";
```

Comment:
I don't understand why this was necessary. Just so we can add comments?

Reply:
It was necessary to make it clear, that dbdata is not a plain text file, but a properties file. Else the next reader wonders why the comment header work correctly or where it gets stipped out. For a properties file this is correct by definition.